wagtail/contrib/redirects/migrations/0001_initial.py

```diff
@@ -24,7 +24,7 @@ class Migration(migrations.Migration):
                     "old_path",
                     models.CharField(
                         verbose_name="Redirect from",
-                        max_length=255,
+                        max_length=2000,
```

Comment:
please don't edit existing migrations.
The changes in this PR bump up the max_length and this is what `0009_alter_redirect_old_path.py` is for

Reply:
@zerolab I made the migration. But like I said, this is just an initial PR. Not super sure it is the right course of action. But there is a discussion about it in the issue that led to it #11053, if you have some time and could check through it.

Reply:
@Temidayo32 What @zerolab is saying is that we must never modify migrations that were already applied to production databases, otherwise it messes up the history of what happened to the models over time, leading to all sorts of errors.

But it can be tricky to remember which migrations were applied to which environment. In our case here, `redirects/migrations/0001_initial.py` was already applied to every Wagtail project using the `redirects` app, so modifying it will do nothing to the database of existing projects, but it will confuse Django which will think that a limit of 2000 characters was already applied to `old_path`. So when Django will try to apply migration `0009_alter_redirect_old_path`, it will think that the 2000 characters limit is already applied and will do nothing. It's later that we will find the issue and good luck for following exactly what happened and fix it.

As you see, it can be complicated to follow how migrations work and which ones were already applied in which environment, so the common practice is to **never modify migrations that were already committed**.

Reply:
@BertrandBordage  Oh, I see. Thanks for the clarifications

Reply:
@BertrandBordage @zerolab removing the migration file would mean there would now be a failing tests with `test_migration`. Can the PR be merged still?